src/goto.jl

```diff
@@ -0,0 +1,84 @@
+handle("goto") do data
+  @destruct [
+    word,
+    path || "",
+    mod || "Main",
+    column || 1,
+    row || 1,
+    startRow || 0,
+    context || "",
+    onlytoplevel || false
```

Comment:
`onlytoplevel` gonna be `true` when it's called from the doc pane, where there is no local context